yt_dlp/extractor/playsuisse.py

```diff
@@ -135,11 +138,29 @@ class PlaySuisseIE(InfoExtractor):
             url
         }'''
 
+    def _perform_login(self, username, password):
+        if username is None or password is None:
+            self.raise_login_required('Your Play Suisse account\'s details must be provided with --username and --password.')
+
+        login_page = self._download_webpage('https://www.playsuisse.ch/api/sso/login?x=x&locale=de&redirectUrl=https%3A%2F%2Fwww.playsuisse.ch%2F', None, note="Downloading login page")
+        settings_str = re.search("^var SETTINGS(.*)", login_page, flags=re.MULTILINE).group()[len("var SETTINGS = "):-2]
+        settings = json.loads(settings_str)
```

Comment:
use `_search_json()`
```suggestion
        settings = self._search_json(r'var\s+SETTINGS\s*=', login_page, 'settings', None)
```